_CodingChallenges/003-snake-game-p5.md

```diff
@@ -42,6 +42,13 @@ contributions:
       name: "Makarand Subhash Lahane"
       url: "https://mklahane.github.io/Game-Development/"
     url: "https://mklahane.github.io/Game-Development/NewSnakeGame"
+    
```

Comment:
The tests are failing because there is some whitespace here. As soon as this is fixed, it should pass! 👍